From 74122c2c00237ec3819e7195c3f7113d2c608f37 Mon Sep 17 00:00:00 2001
From: Balaji Selvanathan <quic_bselvana@quicinc.com>
Date: Fri, 21 Feb 2025 12:22:41 +0530
Subject: [PATCH] PENDING: Changes to enable Verified Boot for Lemans in U-Boot

Necessary/missing configs are added in configs/qcom_defconfig file to enable Verified Boot for Lemans in U-Boot

Signed-off-by: Balaji Selvanathan <quic_bselvana@quicinc.com>
Upstream-Status: Pending
---
 configs/qcom_defconfig | 12 ++++++++++++
 1 file changed, 12 insertions(+)

diff --git a/configs/qcom_defconfig b/configs/qcom_defconfig
index f71a2472c28..856824ac081 100644
--- a/configs/qcom_defconfig
+++ b/configs/qcom_defconfig
@@ -158,3 +158,15 @@ CONFIG_FASTBOOT_OEM_RUN=y
 CONFIG_FASTBOOT_CMD_OEM_CONSOLE=y
 CONFIG_FASTBOOT_OEM_BOARD=y
 CONFIG_FASTBOOT_UUU_SUPPORT=y
+
+CONFIG_RSA=y
+CONFIG_ECDSA=y
+CONFIG_FIT_SIGNATURE=y
+CONFIG_FIT_RSASSA_PSS=y
+CONFIG_FIT_CIPHER=y
+CONFIG_ECDSA_VERIFY=y
+CONFIG_AES=y
+CONFIG_RSA_VERIFY_WITH_PKEY=y
+CONFIG_ASYMMETRIC_KEY_TYPE=y
+CONFIG_SHA512=y
+CONFIG_SHA384=y
-- 
2.34.1

